### DISCUSSION TITLE
Is the project dead ?

### DISCUSSION THREAD

**Discussion by Rootax** [arvidn/libtorrent#7544] [General]
Hi,

Given the lack of recent commits, and that arvidn seems focus on other stuffs/github repos, can we hope to have libtorrent 2.1.x one day (with the i/o stack reworked), oooooor this is it for libt ? If it's dead, don't get me wrong is was an amazing run, but I'm just curious. 

Thx you :)

**Comment by arvidn** [comment]
I intend to get back to libtorrent soon. The disk I/O has been swallowing a lot of time

**Reply by Rootax** [reply]
It's nice to read :)

**Reply by master255** [reply]
It's normal for a developer to take a bit of a break from their project. I do that sometimes, too.
Then it's more interesting to come back to it.

**Comment by master255** [comment]
**I think we should rejoice!** The library has been through a lot of challenges and has done very well.
The technology in this library has reached its peak.
**Now that we have such a good library we should use it.**

Arvid will probably make some additions, but these are not radical changes.

The main goal of this library has been achieved.
It is almost impossible to make something better!
